README.md

```diff
@@ -33,8 +33,13 @@ sudo pacman -S gcc make sdl2
 If your Linux distribution is immutable (such as Fedora Atomic, Bazzite or SteamOS), you can use [Distrobox](https://distrobox.it/). 
 
 #### Windows
+Visual Studio 2022 or later with the C++ and C development tools installed
```

Comment:
```suggestion
Visual Studio 2022 or later with the C++ and C development tools installed and vcpkg integration is required, if vcpkg is installed separately and not from Visual Studio Installer it can be integrated using the following command:
```